### PR TITLE
Fix reload bar not disappearing on console settings page.

### PR DESCRIPTION
**Pull request by adfost**
Fixes reload bar not disappearing on console settings page.

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]
@adfost can you fix the title of the PR?

**Comment by adfost** [comment]
> @adfost can you fix the title of the PR?

oh I thought I already did, just changed it.

**Comment by dvaldivia** [comment]
I got an error 502 and still the bar didn't dismiss

Screen Shot 2021-11-01 at 10 23 51 PM

**Comment by dvaldivia** [comment]
never mind, I had to make assets before building a new container image to test on kubernetes, works

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Console.tsx [hunk 1]
```diff
@@ -211,6 +211,10 @@ const Console = ({
         serverNeedsRestart(false);
       })
       .catch((err: ErrorResponseHandler) => {
+        serverIsLoading(false);
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Console.tsx:214]: you are already calling this on line 218

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Console.tsx (hunk 1 above)
Comment (adfost) [reply portal-ui/src/screens/Console/Console.tsx]: fixed

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**